src/ScheduledTasks/Client/ScheduleHandle.cs

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Client.Entities;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DurableTask.ScheduledTasks;
```

Comment:
TODO: need to provide customer operation polling functions or wrap function to poll for resource(schedule) reaching target status functions or both. either way need to give customer failure details if it failed. still testing to find out what returned from get entityinstance and how it captures the operation failures to decide